### DISCUSSION TITLE
Everything is done but links aren't working

### DISCUSSION THREAD

**Discussion by Mokshgrover22** [microsoft/SATechnicalOnboarding#88] [Questions]
So I have done everything which was necessary to complete the technical onboarding process, the issue I'm facing is that I have made my portfolio but the link isn't opening on other window

**Comment by AnthonyByansi** [comment]
hi @Mokshgrover22

All you need to do is Install the Azure static web apps extension for VSCode. To view the website in the browser, _right-click the project in the Static Web Apps extension, and select Browse Site._

![extension-browse-site](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/3a15ac8a-bad7-497b-badc-e67826b9fcf3)

If you've already deployed your Static web app on Azure, please take note that deployments must be done from the main branch just like so;
![quickstart-portal-source-control](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/f3c4aa15-bf71-4723-850a-54cb8175d5d9)
